### ISSUE TITLE
Why can not delete experiments and runs?

### ISSUE THREAD

**Issue by hackenzheng** [kubeflow/pipelines#441]
The pipeline can been deleted. But the experiment and run can't. How can delete?

**Comment by yebrahim** [comment]
Experiments should be probably be deletable, but not runs. Runs are meant to be repeatable units that document the user's experimentation. We should still provide some sort of archiving mechanism for clean up though.

**Comment by Ark-kun** [comment]
I agree with Yasser. While experiments may be deleted, runs should be preserved. We might allow to hide them in future though.

**Comment by hackenzheng** [comment]
okay... In addition, it maybe a demand to delete runs to clear up workflows on k8s. Otherwise the GPU resources can't be released.

**Comment by RakeshRaj97** [comment]
Experiments can be deleted programmatically using `delete_experiment(experiment_id='')` 

List the experiments using `.list_experiments(namespace='')`
